docs/other/performance.md

```diff
@@ -0,0 +1,11 @@
+## Improving composer build performance
```

Comment:
Yeah, it would be good to also at least point people to notes on Windows performance, NFS/rsync/native share performance, how to adjust memory limits (for VM and php especially), etc. Also, cachier, things like that. Simple is best, but anything that people might not ordinarily know about would be good to document.